prov/efa/test/README.md

```diff
@@ -0,0 +1,53 @@
+# EFA unit tests
+
+## How to run
+
+To run efa unit tests, you will need to have cmocka installed.
+See: https://github.com/clibs/cmocka/blob/master/INSTALL.md
```

Comment:
I'm not sure if you should be pointing to this repo - hasn't been updated since 2015 whereas the files on the website for cmocka.org have a version created in 2019

Reply:
done